src/tests/test_salute.py

```diff
@@ -0,0 +1,35 @@
+import os
+import json
+import pytest
+
+from ..service.salute import SaluteService
+
+@pytest.fixture(name="salute", autouse=True)
+def fixture_salute():
+    return SaluteService()
+
+@pytest.fixture(name="file_salute", autouse=True)
+def fixture_file_salute():
+    filename = os.path.join(os.path.dirname(__file__), "../json/salutes.json")
+    with open(file=filename, encoding="utf-8") as f:
+        data = f.read()
+        return json.loads(data)['salutes']
```

Comment:
This can be simplified
```suggestion
        return json.load(f)['salutes']
```